### PR TITLE
@ #379 | should update teracy-dev development guide

### PR DESCRIPTION
**Pull request by hieptranquoc**
@hoale @phuonglm please help me check this PR! 

connect #379

### CODE REVIEW

**Comment by hoatle** [comment]
@hieptranquoc make sure the doc should not duplicate anything from #407

**Comment by hieptranquoc** [comment]
@hoale I have upgraded, now there is no duplicate. please help me check this PR!

**Comment by hoatle** [comment]
@phuonglm please review then

File: docs/teracy_dev_development.rst [hunk 1]
```diff
@@ -162,6 +169,8 @@ We're going to use Bento_ to build base boxes.
         $ git clone git@github.com:chef/bento.git
         $ cd bento
 
+    Note: You may encounter some errors about `public key`. Do not worry, you should add RSA key to your github and clone project again.
```
Comment (phuonglm) [inline docs/teracy_dev_development.rst]: use https://github.com/chef/bento.git instead

File: vagrant_config.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     ]
   },
   "vm": {
-    "hostname": "teracy.dev",
+    "hostname": "teracy-dev.dev",
```
Comment (phuonglm) [inline vagrant_config.json]: do we need to change host name?

**Review by phuonglm — COMMENTED** [review]
@hieptranquoc please update with my comment

File: vagrant_config.json (hunk 2 above)
Comment (hieptranquoc) [reply vagrant_config.json]: fixed!

**Comment by hieptranquoc** [comment]
@phuonglm i have updated, please help me check this PR again!

File: docs/teracy_dev_development.rst (hunk 1 above)
Comment (hieptranquoc) [reply docs/teracy_dev_development.rst]: updated!

**Review by phuonglm — DISMISSED** [review]

**Comment by hoatle** [comment]
assigned to @hoavt

File: docs/teracy_dev_development.rst [hunk 3]
```diff
@@ -1,44 +1,31 @@
...
-- register an account at hub.docker.com
-- enable `teracy-dev` repository on travis-ci.org
-- add travis-ci settings with these variables:
-    + ``DOCKER_USERNAME``: Fill with your Docker username account
-    + ``DOCKER_PASSWORD``: Fill with your Docker password account
-    + ``DEV_DOCKER_IMAGE``: Fill with `<your_docker_username>/teracy-dev`
-    + ``DOCS_DOCKER_IMAGE``: Fill with `<your_docker_username>/teracy-dev-docs`
-    + ``GH_REPO``: Fill with `<your_github_username>/teracy-dev>` to git push on your repo
-    + ``GH_TOKEN``: Fill with `<your_github_personal_access_token>` to git push on your repo
+Please follow document at https://github.com/teracyhq/dev/tree/develop/dev-setup/README.md#travis-ci-configuration to set up CI system on travis-ci.org :
```
Comment (hoavt) [inline docs/teracy_dev_development.rst]: Please follow the document at

File: docs/teracy_dev_development.rst [hunk 4]
```diff
@@ -67,6 +60,11 @@ Rake Tasks
 ..  code-block:: bash
 
     $ docker-compose run --rm dev rake build
+
+You will see similiar like this:
```
Comment (hoavt) [inline docs/teracy_dev_development.rst]: You will see something similar to this:

File: docs/teracy_dev_development.rst [hunk 5]
```diff
@@ -78,28 +76,35 @@ Rake Tasks
...
     0 examples, 0 failures
 
+
 - ``$ rake berks_install`` to install vendor cookbooks with Berkshelf_.
 
 ..  code-block:: bash
 
     $ docker-compose run --rm dev rake berks_install
+
+You will see similiar like this:
```
Comment (hoavt) [inline docs/teracy_dev_development.rst]: You will see something similar to this:

File: docs/teracy_dev_development.rst [hunk 6]
```diff
@@ -123,16 +128,18 @@ We're going to use Bento_ to build base boxes.
     ..  code-block:: bash
 
         $ cd /tmp
-        $ wget http://download.virtualbox.org/virtualbox/4.3.20/virtualbox-4.3_4.3.20-96996~Ubuntu~precise_amd64.deb
-        $ sudo dpkg -i virtualbox-4.3_4.3.20-96996~Ubuntu~precise_amd64.deb
+        $ wget http://download.virtualbox.org/virtualbox/5.1.10/virtualbox-5.1_5.1.10-112026~Ubuntu~xenial_amd64.deb
+        $ sudo dpkg -i virtualbox-5.1_5.1.10-112026~Ubuntu~xenial_amd64.deb
         $ sudo apt-get install -f -y
 
+    Note: You may encounter some errors. Do not worry, you should install the missing dependencies. After install, you run `/sbin/vboxconfig` as root.
```
Comment (hoavt) [inline docs/teracy_dev_development.rst]: After installing, run ...

File: docs/teracy_dev_development.rst [hunk 7]
```diff
@@ -152,16 +159,18 @@ We're going to use Bento_ to build base boxes.
...
 #.  `Bento` Repository Clone
 
     ..  code-block:: bash
 
         $ ws
-        $ git clone git@github.com:chef/bento.git
+        $ git clone https://github.com/chef/bento.git
         $ cd bento
 
+    Note: You may encounter some errors about `public key`. Do not worry, you should add RSA key to your github and clone project again.
```
Comment (hoavt) [inline docs/teracy_dev_development.rst]: add the RSA key... and clone the project again.

**Review by hoavt — COMMENTED** [review]
You could install any versions of Virtualbox that matched the host machine. --> You could install any versions of Virtualbox that matches the host machine.


@hieptranquoc Please check my comments

**Comment by hieptranquoc** [comment]
@hoavt updated! please help me check this PR again!

**Review by hoavt — APPROVED** [review]

**Comment by hoavt** [comment]
It's ok to merge

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**